src/web/jcc/client/components/CommandControl.tsx

```diff
@@ -801,6 +807,70 @@ export default class CommandControl extends React.Component {
 		}
 	}
 
+	
+	/**
+	 * Use JaiaAPI to poll the botPath data.
+	 */
+	pollBotPaths() {
+		console.log(this.lastBotPathPointUtime)
```

Comment:
This console.log()  gunks up the console a good bit

Reply:
Removed!